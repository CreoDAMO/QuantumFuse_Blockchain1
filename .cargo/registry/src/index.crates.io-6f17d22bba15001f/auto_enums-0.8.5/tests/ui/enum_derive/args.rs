// SPDX-License-Identifier: Apache-2.0 OR MIT

use auto_enums::enum_derive;

#[enum_derive(Clone, =>)] //~ ERROR expected identifier
enum Enum1<A, B> {
    A(A),
    B(B),
}

#[enum_derive(foo::bar::!)] //~ ERROR expected identifier
enum Enum2<A, B> {
    A(A),
    B(B),
}

#[enum_derive(Clone, Foo:)] //~ ERROR expected `,`
enum Enum3<A, B> {
    A(A),
    B(B),
}

#[enum_derive(Clone Foo)] //~ ERROR expected `,`
enum Enum4<A, B> {
    A(A),
    B(B),
}

fn main() {}
