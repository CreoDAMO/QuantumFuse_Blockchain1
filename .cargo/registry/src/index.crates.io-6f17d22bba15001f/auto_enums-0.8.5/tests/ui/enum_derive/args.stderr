error: expected identifier
 --> tests/ui/enum_derive/args.rs:5:22
  |
5 | #[enum_derive(Clone, =>)] //~ ERROR expected identifier
  |                      ^

error: expected identifier
  --> tests/ui/enum_derive/args.rs:11:25
   |
11 | #[enum_derive(foo::bar::!)] //~ ERROR expected identifier
   |                         ^

error: expected `,`
  --> tests/ui/enum_derive/args.rs:17:25
   |
17 | #[enum_derive(Clone, Foo:)] //~ ERROR expected `,`
   |                         ^

error: expected `,`
  --> tests/ui/enum_derive/args.rs:23:21
   |
23 | #[enum_derive(Clone Foo)] //~ ERROR expected `,`
   |                     ^^^
